src/client/src/index.css

```diff
@@ -4,4 +4,9 @@ html, body {
   font-family: "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue", sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  height: 100%;
+} 
+
+#root {
+  height: 100%;
```

Comment:
Instead of having to modify the root, I think you can use `100vh` on `.loading-outer`.